config/settings/test.yml

```diff
@@ -204,6 +204,13 @@ sidekiq:
   github_oauth_secret: 000xxx
 
 check_in:
+  travel_reimbursement_api:
+    auth_url: https://login.microsoftonline.us
```

Comment:
Do you think we can have a separate section for auth & claims api?

Reply:
My thinking behind this was to keep the settings and code for claim submission in one place.